lib/msal-browser/src/config/Configuration.ts

```diff
@@ -2,14 +2,14 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { AuthOptions, SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
+import { SPAAuthOptions, SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
 import { BrowserUtils } from "../utils/BrowserUtils";
 import { BrowserConstants } from "../utils/BrowserConstants";
 
 // Default timeout for popup windows in milliseconds
 const DEFAULT_POPUP_TIMEOUT_MS = 60000;
 
-export type BrowserAuthOptions = AuthOptions & {
+export type BrowserAuthOptions = SPAAuthOptions & {
```

Comment:
What is the difference between SPA Auth and Browser Auth?

Reply:
SPA Auth is from common, browser auth is from msal-browser. @sameerag does this need to be separate from node auth config?

Reply:
For now yes, it isn't generic enough to reuse for node. I will work on cleaning and integrating SPA code in msal-common once we have node working end to end